Dockerfile.build-tools

```diff
@@ -112,6 +112,40 @@ RUN for package in Capture::Tiny DateTime Devel::Cover Digest::MD5 File::Spec JS
     && make install \
     && rm -rf ../lcov.tar.gz
 
+# Compile and install the static OpenSSL library
+ENV OPENSSL_VERSION=3.2.2
```

Comment:
It might have been discussed somewhere, but why do we use openssl 3.2.2? Bookworm comes with [3.0](https://packages.debian.org/bookworm/openssl), and the latest openssl version is [3.3](https://www.openssl.org/news/openssl-3.3-notes.html).

Reply:
This is straight from @rahulinux 's original patch. @rahulinux , what was your rationale at the time?

Reply:
We would want to use exactly what bullseye uses, that's the theme in this PR.

That would be `1.1.1w`.

However, bullseye has some patches on top of vanilla `1.1.1w`: https://sources.debian.org/patches/openssl/1.1.1w-0+deb11u1/

AFAICT the only relevant patch is https://sources.debian.org/patches/openssl/1.1.1w-0%2Bdeb11u1/Set-systemwide-default-settings-for-libssl-users.patch/

Reply:
For the record, openssl doesn't really matter for production Pageservers at runtime. We could use artifacts built `--with-openssl=no` and be fine.

However, for neon_local setups (including tests in CI), we use the same Postgres build for Computes as we do for Pageserver (`./pg_install/`).

Reply:
Will turn this into a follow-up.

Reply:
https://github.com/neondatabase/cloud/issues/14526